content/en/docs/ops/configuration/telemetry/envoy-stats/index.md

```diff
@@ -21,30 +21,62 @@ To see the statistics for a pod:
-for an explanation of the data recorded.
+Envoy generates statistics about its behavior, scoping the statistics by proxy function. Examples include:
 
-By default, Istio configures Envoy to record minimal statistics.  The default collection
+- [Upstream connection](https://www.envoyproxy.io/docs/envoy/latest/configuration/upstream/cluster_manager/cluster_stats)
+- [Listener](https://www.envoyproxy.io/docs/envoy/latest/configuration/listeners/stats)
+- [HTTP Connection Manager](https://www.envoyproxy.io/docs/envoy/latest/configuration/http/http_conn_man/stats)
+- [TCP proxy](https://www.envoyproxy.io/docs/envoy/latest/configuration/listeners/network_filters/tcp_proxy_filter#statistics)
+- [Router](https://www.envoyproxy.io/docs/envoy/latest/configuration/http/http_filters/router_filter.html?highlight=vhost#statistics)
+
+By default, Istio configures Envoy to record a minimal set of statistics to reduce the overall CPU and memory footprint of the installed proxies. The default collection
 keys are:
 
 - `cluster_manager`
 - `listener_manager`
 - `server`
 - `cluster.xds-grpc`
+- `wasm`
 
 To see the Envoy settings for statistics data collection use
 [`istioctl proxy-config bootstrap`](/docs/reference/commands/istioctl/#istioctl-proxy-config-bootstrap) and follow the
 [deep dive into Envoy configuration](/docs/ops/diagnostic-tools/proxy-cmd/#deep-dive-into-envoy-configuration).
 Envoy only collects statistical data on items matching the `inclusion_list` within
 the `stats_matcher` JSON element.
 
-To Configure Envoy to record statistics for inbound or outbound traffic, add the
-`sidecar.istio.io/statsInclusionPrefixes` annotation to the pod template in the Kubernetes `Deployment`.
-Add the `cluster.outbound` prefix to gather data about outbound traffic activity and circuit breaking.
-To gather data on inbound traffic, add the `listener` prefix.  The sample
-[fortio-deploy.yaml]({{< github_file>}}/samples/httpbin/sample-client/fortio-deploy.yaml)
-shows use of `sidecar.istio.io/statsInclusionPrefixes` with the `cluster.outbound` prefix.
+{{< tip >}}
+Note: The names of Envoy statistics can vary based on the composition of Envoy configuration. As a result, the exposed names of statistics for Envoys managed by Istio are subject to the configuration behavior of Istio.
+If you build or maintain dashboards or alerts based on Envoy statistics, it is **strongly recommended** to examine the
+statistics in a canary environment **before upgrading Istio**.
+{{< /tip >}}
+
+To configure Istio proxy to record additional statistics, you can add [`ProxyConfig.ProxyStatsMatcher`](/docs/reference/config/istio.mesh.v1alpha1/#ProxyStatsMatcher) to your mesh config. For example, to enable stats for circuit breaker, retry, and upstream connections globally, you can specify stats matcher as follow:
+
+{{< tip >}}
+Proxy needs to restart to pick up the stats matcher configuration.
```

Comment:
do users need to manually restart? if yes i would specify

Reply:
I am not sure what is categorized as `manual restarted`. Any restart of pod will pick up the configuration. This is just to set up expectation to the user that this won't take effect at runtime.